test/e2e/atlasmap_test.go

```diff
@@ -113,6 +116,42 @@ func atlasMapDeploymentTest(t *testing.T, f *framework.Framework, ctx *framework
 		}
 		scheme = "https"
 		host = atlasMapRoute.Spec.Host
+
+		//verify Console Link NamespaceDashboard
+		openShiftSemVer := util.GetClusterVersionSemVer(f.KubeConfig)
```

Comment:
There's another potential nil pointer exception here. `GetClusterVersionSemVer` has:

```go
if errors.IsNotFound(err) {
	// default to OpenShift 3 as ClusterVersion API was introduced in OpenShift 4
	openShiftSemVer, _ = semver.NewVersion("3")
} else {
	return nil
}
```

So if an error occurs that is not trapped by the `IsNotFound` condition, `nil` is returned and is not handled by the caller. I'm actually seeing this happen on a local Minishift cluster:

```
$ eval $(minishift docker-env)

$ make build

$ make install

$ make deploy

$ oc logs atlasmap-operator-7cdb8b594f-f2t9w
{"level":"info","ts":1585821466.9990215,"logger":"cmd","msg":"Go Version: go1.13.5"}
{"level":"info","ts":1585821466.9990773,"logger":"cmd","msg":"Go OS/Arch: linux/amd64"}
{"level":"info","ts":1585821466.9990823,"logger":"cmd","msg":"Operator SDK Version: v0.15.1"}
{"level":"info","ts":1585821466.9990854,"logger":"cmd","msg":"AtlasMap Operator Version: 0.3.0"}
{"level":"info","ts":1585821466.9990885,"logger":"cmd","msg":"AtlasMap Operator Git Commit: 3c1271f0"}
{"level":"info","ts":1585821466.9990923,"logger":"cmd","msg":"AtlasMap Default Image: docker.io/atlasmap/atlasmap:latest"}
{"level":"info","ts":1585821466.9992895,"logger":"leader","msg":"Trying to become the leader."}
{"level":"info","ts":1585821467.6745458,"logger":"leader","msg":"Found existing lock with my name. I was likely restarted."}
{"level":"info","ts":1585821467.674589,"logger":"leader","msg":"Continuing as the leader."}
{"level":"info","ts":1585821468.332753,"logger":"controller-runtime.metrics","msg":"metrics server is starting to listen","addr":"0.0.0.0:8383"}
{"level":"info","ts":1585821468.3341796,"logger":"cmd","msg":"Registering Components."}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x53b192]
goroutine 1 [running]:
github.com/Masterminds/semver.(*Version).Prerelease(...)
	/home/james/go/pkg/mod/github.com/!masterminds/semver@v1.5.0/version.go:146
github.com/Masterminds/semver.constraintGreaterThanEqual(0x0, 0xc000403980, 0xc0001df1a0)
	/home/james/go/pkg/mod/github.com/!masterminds/semver@v1.5.0/constraints.go:289 +0x22
github.com/Masterminds/semver.(*constraint).check(...)
	/home/james/go/pkg/mod/github.com/!masterminds/semver@v1.5.0/constraints.go:176
github.com/Masterminds/semver.Constraints.Check(0xc000115380, 0x1, 0x1, 0x0, 0x0)
	/home/james/go/pkg/mod/github.com/!masterminds/semver@v1.5.0/constraints.go:49 +0x82
github.com/atlasmap/atlasmap-operator/pkg/controller/atlasmap.newOperatorActions(0x18728a0, 0xc0002669c0, 0x188b2e0, 0xc00045e900, 0xc0000b3c20, 0x0, 0x0)
	atlasmap-operator/pkg/controller/atlasmap/action.go:44 +0x872
github.com/atlasmap/atlasmap-operator/pkg/controller/atlasmap.add(0x188b2e0, 0xc00045e900, 0x182ce00, 0xc000249050, 0x0, 0x0)
	atlasmap-operator/pkg/controller/atlasmap/atlasmap_controller.go:114 +0x538
github.com/atlasmap/atlasmap-operator/pkg/controller/atlasmap.Add(0x188b2e0, 0xc00045e900, 0xb50504, 0x233a5b0)
	atlasmap-operator/pkg/controller/atlasmap/atlasmap_controller.go:48 +0x17c
github.com/atlasmap/atlasmap-operator/pkg/controller.AddToManager(0x188b2e0, 0xc00045e900, 0x0, 0x0)
	atlasmap-operator/pkg/controller/controller.go:13 +0x79
```

Reply:
fixed